google-cloud-bigquery/src/main/java/com/google/cloud/bigquery/Acl.java

```diff
@@ -568,9 +570,82 @@ Access toPb() {
     }
   }
 
+  /** Expr represents the conditional information related to dataset access policies. */
+  public static final class Expr implements Serializable {
+    // Textual representation of an expression in Common Expression Language syntax.
+    private final String expression;
+    /**
+     * Optional. Title for the expression, i.e. a short string describing its purpose. This can be
+     * used e.g. in UIs which allow to enter the expression.
+     */
+    private final String title;
+    /**
+     * Optional. Description of the expression. This is a longer text which describes the
+     * expression, e.g. when hovered over it in a UI.
+     */
+    private final String description;
+    /**
+     * Optional. String indicating the location of the expression for error reporting, e.g. a file
+     * name and a position in the file.
+     */
+    private final String location;
+
+    public Expr(String expression, String title, String description, String location) {
```

Comment:
Are all the parameters needed to form an valid Expr?
E.g., Is it valid to for users to create an Expr with no description, location, title? (It seems like the answer is no from the parameter descriptions)
If that is the case, then providing a builder might be preferable.

Reply:
I think only the expression is required for a valid `Expr`. I have implemented a builder for the class but have kept the public constuctor for `Expr` for ease of use.